detox/src/android/espressoapi/EspressoDetox.js

```diff
@@ -52,7 +52,7 @@ class EspressoDetox {
   }
 
   static setURLBlacklist(urls) {
-    if (typeof urls !== 'object' || !urls instanceof Array) {
+    if (typeof urls !== 'object' || !Array.isArray(urls)) {
```

Comment:
I don't see that it is tested anywhere currently, so I didn't add an extra test here, but anyway the `runScriptWithArgs` covers it, so I hope it's okay.